MAUI/Button/Customization.md

```diff
@@ -124,6 +124,34 @@ button.VerticalTextAlignment = TextAlignment.Center;
 {% endhighlight %}
 {% endtabs %}
 
+## Text Transform
+
+Users can now customize the [`SfButton`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfButton.html) text using the `TextTransform` property. They can easily switch between uppercase, lowercase, none, or default styles.
```

Comment:
Users can now customize the [`SfButton`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfButton.html) text casing using the `TextTransform` property. They can easily switch between uppercase, lowercase, none, or default casing.

Reply:
content updated